framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -20,6 +20,9 @@
     /** The upper bound of the interval, inclusive. */
     public final long to;
 
+    /** Should this range take overflow into account or ignore it? */
+    public static boolean IGNORE_OVERFLOW = false;
```

Comment:
I changed this to a static field rather, so that it doesn't have to be passed around everywhere.  Let me if you disagree with this change.